package.json

```diff
@@ -8,11 +8,14 @@
     "lint": "git diff HEAD --name-only --diff-filter=ACM | grep '.js$' | xargs node ./node_modules/eslint/bin/eslint.js --quiet",
     "lint-full": "node ./node_modules/eslint/bin/eslint.js .",
     "test": "jest --config ./config/jest/config.js",
+    "test:runner": "cypress open --project config/cypress --env name=runner",
```

Comment:
I wonder if this script should also start the local dev server, so that you don't need to manually run the server using `npm run docs`. Not to mention, even if you did run the docs it will still fail because the cypress config script is set to visit localhost on port `8001`, whereas doing `npm run docs` without setting `PORT` will start the server on port `8000`.